jormungandr/src/network/service.rs

```diff
@@ -299,4 +300,12 @@ impl GossipService for NodeService {
             logger,
         )
     }
+
+    fn peers(&mut self) -> Self::PeersFuture {
+        intercom::unary_future(
+            self.channels.client_box.clone(),
+            self.logger().new(o!("request" => "Peers")),
+            ClientMsg::GetPeers,
+        )
```

Comment:
Could have just used `global_state.topology`, no need to involve the client task in bouncing messages around and sharing the topology state.

Reply:
I followed what was done in GetTip, and I agree it seems overkill to do it this way